studio/README.md

```diff
@@ -1,3 +0,0 @@
-# Studio Helm Chart
```

Comment:
Please don't delete it,  it is welcome information after opening the repository. :-)

Reply:
The README file was moved to the project root. `studio` directory only contains the chart now.